pkg/operators/kubenetworkmanager/kubenetworkmanager.go

```diff
@@ -0,0 +1,295 @@
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubenetworkmanager
```

Comment:
Could you add some go-documentation for this package?

```
// Package kubeipresolver provides an operator that enriches events by looking
// up IP addresses in Kubernetes resources such as pods and services. It is
// useful for the 'trace network' gadget. The operator is only enabled on
// gadgets that use this feature (checked by CanOperateOn), so it does not have
// a performance impact on other gadgets.
package kubeipresolver
```

Reply:
I like your name a lot more. Generally we will probably rename some network-related operators or so when we start working on #996

Reply:
Sounds fine but maybe change the last sentence to:
```
// Package kubeipresolver provides an operator that enriches events by looking
// up IP addresses in Kubernetes resources such as pods and services. It is
// currently only used by the 'trace network' gadget.
```